### PR TITLE
Battery change

### PR DESCRIPTION
**Pull request by anatoly-particle**

### CODE REVIEW

**Review by monkbroc — APPROVED** [review]
Looks good

File: .spelling [hunk 1]
```diff
@@ -231,6 +232,7 @@ PBDE
 PTCRB
 GCF
 000mAh
+800mAh
```
Comment (stefan-moraru) [inline .spelling:235]: 800?

**Review by stefan-moraru — APPROVED** [review]
👍

File: .spelling (hunk 1 above)
Comment (anatoly-particle) [reply .spelling:235]: Yeah, that for spelling check _after_ comma: “1,800mAh”.

**Pull request merged**